src/common/clrpickercmn.cpp

```diff
@@ -64,12 +67,19 @@ bool wxColourPickerCtrl::Create( wxWindow *parent, wxWindowID id,
                                         wxDefaultPosition, wxDefaultSize,
                                         GetPickerStyle(style));
 
+
     // complete sizer creation
     wxPickerBase::PostCreation();
 
     m_picker->Bind(wxEVT_COLOURPICKER_CHANGED,
             &wxColourPickerCtrl::OnColourChange, this);
 
+    m_picker->Bind(wxEVT_COLOUR_SELECTED,
```

Comment:
Whitespace nitpick: The second-line indentation for Bind()ing the two new events does not match indentation above?